pkg/api/tests.go

```diff
@@ -209,6 +216,12 @@ func BuildTestsResults(dbc *db.DB, release, period string, collapse, includeOver
 		rawQuery = rawFilter.ToSQL(rawQuery, apitype.Test{})
 	}
 
+	// should we filter down to only tests on the watchlist or not?
+	if watchlist {
+		log.Info("WATCHLIST REQUESTED")
+		rawQuery = rawQuery.Where("watchlist = ?", true)
+	}
```

Comment:
Could this not just be part of a filter?

Reply:
Ooo yes, did not even think of that. I'll change it, thanks.